### DISCUSSION TITLE
early game item availability

### DISCUSSION THREAD

**Discussion by lstebner** [jeremyckahn/farmhand#82] [Ideas]
I would like to have more items available earlier on (seeds mainly). the early game involves a lot of carrots currently, for the first 30 days or so. then spinach is available, and maybe a cow in the first 60. this is a pretty long time. most other farming sim type games consider 30ish days to be a "month" of time, and 60 to be half a year. having more variety would cause the player to think about managing their space more, and more seeds would lead to more recipes earlier on too. i think it would all just add a bit more freshness as the player is learning the game.

**Comment by jeremyckahn** [comment]
This is a really good point. Rather than giving access to later-game crops earlier, I wonder if we could have a special tier of crops, one of which that is randomly available to a player when they create a new game? That would make individual game experiences more unique, and if we ever implement multiplayer item trading, it could create some interesting scarcity dynamics.

This is more work of course, but I think it'll make the game more interesting in the long run!
